corehq/apps/es/cases.py

```diff
@@ -21,6 +21,9 @@
 
 class CaseES(HQESQuery):
     index = 'cases'
+    default_filters = {
+        'is_commcare_case': filters.doc_type("CommCareCase"),
```

Comment:
the pillow should remove soft deletions. see [code](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/pillowtop/processors/elastic.py#L32) and [test](https://github.com/dimagi/commcare-hq/blob/master/testapps/test_pillowtop/tests/test_case_pillow.py#L47)

are there lots of `CommCareCase-Deleted` in the index?